packages/obonode/obojobo-chunks-math-equation/editor-component.js

```diff
@@ -36,6 +36,7 @@ class MathEquation extends React.Component {
 		const content = this.props.element.content
 		this.state = this.contentToStateObj(content)
 
+		this.path = ReactEditor.findPath(this.props.editor, this.props.element)
```

Comment:
Caching the path in the constructor may be a little dangerous.  If this component is reused but moved around in the document - It's path will change but not get updated here.